_events/2022/2022-12-DEI-speaker-series-Mars-Lee.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Mars Lee"
+expires: 2022-12-14
+event_date: "December 14, 2022"
+layout: event
+repeated: false
+duration: 60
+category: dei
+time:
+    - - start: 2022-12-14T22:00:00Z
+        end: 2022-12-14T23:30:00Z
```

Comment:
```suggestion
        end: 2022-12-14T21:30:00Z
```

Reply:
I fully admit, I don't understand the whole event time zone thing - but the times listed here and in the text don't seem to match. I'm assuming the text is correct.

Reply:
But these times also show a duration of 90 minutes - so something else might be wrong?

Reply:
Thank you for converting this to the correct Zulu time. The "30" should have been changed to "00" too